python/pyspark/errors/error_classes.py

```diff
@@ -268,6 +268,11 @@
       "Eval type for UDF must be <eval_type>."
     ]
   },
+  "INVALID_UDTF_BOTH_RETURN_TYPE_AND_ANALYZE_STATICMETHOD" : {
+    "message" : [
+      "The UDTF '<name>' is invalid. It has both its return type and the required 'analyze' static method. Please make it have one of either the return type or the 'analyze' static method in '<name>' and try again."
```

Comment:
just to confirm, it fails even if the user-specified return type == the analyzed result.

Reply:
Yes, it fails when a Python UDTF has both `returnType` and `analyze` static method, to avoid confusions.